NineAnimator/Controllers/Settings Scene/TrackingServiceTableViewController.swift

```diff
@@ -467,17 +467,6 @@ extension TrackingServiceTableViewController {
             // references
             _ = session.start()
             authenticationSessionReference = session
-        } else {
```

Comment:
You can probably re-add this, it was crashing on catalyst so I just removed it all then didn't bother adding it back when I added the #available check